server/src/types/user.types.ts

```diff
@@ -13,4 +13,6 @@ type UserSchema = {
 
 type UserDocument = UserSchema & MongoInjectedFields;
 
-export { UserSchema, UserDocument };
+type UserAuthSchema = Omit<UserDocument, 'password' | 'createdAt' | 'updatedAt'>;
```

Comment:
wow what does this `Omit` do?

Reply:
It omits certain fields from a given type haha. so if you wanna create a new type that is just an existing type stripped down by a couple properties, can use Omit!